go-controller/pkg/kubevirt/pod.go

```diff
@@ -470,11 +470,14 @@ func DiscoverLiveMigrationStatus(client *factory.WatchFactory, pod *corev1.Pod)
 
 	targetPod := vmPods[len(vmPods)-1]
 	livingPods := filterNotComplete(vmPods)
+
+	// if there is no living pods, it can be that something has killed
+	// the vm like OOM killer so there is no live migration going on
+	if len(livingPods) == 0 {
+		return nil, nil
```

Comment:
Please bear with me for a sec here ...

In https://github.com/ovn-kubernetes/ovn-kubernetes/blob/0b513c6319e3258309ffbba0e67ce7a5d427b17e/go-controller/pkg/ovn/base_network_controller_secondary.go#L933, we're ensuring that in the scenario of a migration failure, the src pod LSP is "enabled".

We're OK because we're not in a migration right ? i.e. there's no src pod LSP to enable, right ?